sdk/core/System.ClientModel/src/Internal/PipelineMessageSanitizer.cs

```diff
@@ -0,0 +1,157 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace System.ClientModel.Internal;
+
+/// <summary>
+/// TODO.
+/// </summary>
+internal class PipelineMessageSanitizer
```

Comment:
Same comment about tracking to address later with https://github.com/Azure/azure-sdk-for-net/issues/41035